### DISCUSSION TITLE
Actual costs for a hobbyist ?

### DISCUSSION THREAD

**Discussion by Shotman** [TabbyML/tabby#1091] [Q&A] [answered]
Hello, I'm quite interested in Tabby and I'm wondering if anyone has any data on what would be the actual cost to run it as a hobbyist on something like Modal/HF ?
Because right now I pay a Copilot subscription so if the cost per month is roughly the same I don't see what stops me from switching. You knwo for privacy, Microsoft etc etc

**Comment by wsxiaoys** [comment, marked as answer]
Modal provides $30 per month in free credits, offering approximately 50 hours of T4 GPU time. Depending on your coding activity, the cost can range from free to a few low digits.

Conversely, a T4 setup on Modal, configured with proper settings, can easily support a team of around 15 people (e.g., serving DeepseekCoder-1.3B with approximately 4 QPS).

Assuming 12 active hours per day and 25 active days in a month, the cost is roughly $180 per month, making it approximately $10 per person.

**Reply by Shotman** [reply]
Thank you, then I might try for a month and see how it goes then ! 
I didn't want to try that blind and end up with a few hundred in cost just for a test ^^'
